backend/src/nodes/nodes/onnx/upscale_image.py

```diff
@@ -10,14 +11,36 @@
 from ...properties.inputs import OnnxModelInput, ImageInput, TileSizeDropdown
 from ...properties.outputs import ImageOutput
 from ...properties import expression
-from ...utils.auto_split_tiles import parse_tile_size_input, TileSize
+from ...utils.auto_split_tiles import parse_tile_size_input, TileSize, NO_TILING
 from ...utils.onnx_auto_split import onnx_auto_split
 from ...utils.onnx_model import OnnxModel
 from ...utils.onnx_session import get_onnx_session
 from ...utils.utils import get_h_w_c, convenient_upscale
 from ...utils.exec_options import get_execution_options
 
 
+OnnxInputShape = Literal["BCHW", "BHWC"]
```

Comment:
Can we figure out whether the dim orders are HW or WH? We're doing HW here, but when we define input/output in the Convert to ONNX node, we have:
```
dynamic_axes = {
    "input": {0: "batch_size", 2: "width", 3: "height"},
    "output": {0: "batch_size", 2: "width", 3: "height"},
}
```

I have found people reference it both ways, so I'm currently not sure which is technically correct. It *probably* won't matter in terms of functioning, but it would be good to at least be internally consistent.

Reply:
ONNX docs use `b, c, h, w`

example: https://onnx.ai/onnx/operators/onnx__DepthToSpace.html

So I think our exporter is wrong

Reply:
I was using the same logic that the upscaling process uses. If you follow the `change_shape` parameter into `onnx_auto_split`, you'll find that `onnx_auto_split` assume either `BCHW` or `BHWC`. So if WH is also possible, we have some fixing to do.

Reply:
So the PR is gtg @theflyingzamboni?

Reply:
I don't even know what this export you guys were talking about is lol

Reply:
The PyTorch -> onnx node